include/timer/timer_config.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Race Capture Firmware
+ *
+ * Copyright (C) 2016 Autosport Labs
+ *
+ * This file is part of the Race Capture firmware suite
+ *
+ * This is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details. You should
+ * have received a copy of the GNU General Public License along with
+ * this code. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _TIMER_CONFIG_H_
+#define _TIMER_CONFIG_H_
+
+#include "channel_config.h"
+#include "cpp_guard.h"
+
+CPP_GUARD_BEGIN
+
+#define MODE_LOGGING_TIMER_RPM		0
+#define MODE_LOGGING_TIMER_FREQUENCY	1
+#define MODE_LOGGING_TIMER_PERIOD_MS	2
+#define MODE_LOGGING_TIMER_PERIOD_USEC	3
+
+#define TIMER_SLOW	0
+#define TIMER_MEDIUM	1
+#define TIMER_FAST	2
+
+#define TIMER_FILTER_VALUE_AUTO	-1
```

Comment:
Huh?  Where?  You mean between values and define characters?  Those are simply tab characters as is the Linux standard if so.